airflow/www/templates/airflow/graph.html

```diff
@@ -112,8 +112,8 @@
     // Below variables are being used in dag.js
     var tasks = {{ tasks|safe }};
     var task_instances = {{ task_instances|safe }};
-    var getTaskInstanceURL = '{{ url_for("Airflow.task_instances", dag_id='+
-      dag.dag_id + ', execution_date=' + execution_date + ') }}';
+    var getTaskInstanceURL = "{{ url_for('Airflow.task_instances',
+      execution_date=execution_date)}}&dag_id={{dag.dag_id}}";
```

Comment:
the way we are not using the `dag_id` as a parameter in url_for function and only using execution date is kinda confusing. Either we should use both as parameters or skip both. JS does not provide a proper way to use jinja template variables, so I'd suggest to use both variables outside the `url_for` function. 
Something like:

```
    var getTaskInstanceURL = "{{ url_for('Airflow.task_instances') }}" +
                            "?dag_id=" + encodeURIComponent(dag_id) +
                            "&execution_date=" + encodeURIComponent(execution_date);
```

Reply:
Updated as per your suggestion